packages/editor-ui/src/components/ParameterInput.vue

```diff
@@ -787,10 +787,13 @@ export default mixins(
 		isSecretParameter(): boolean {
 			return this.getArgument('password') === true;
 		},
+		remoteParameterOptionsKeys(): string[] {
+			return (this.remoteParameterOptions || []).map((o) => o.name);
+		},
 	},
 	methods: {
 		isRemoteParameterOption(option: INodePropertyOptions) {
-			return this.remoteParameterOptions.map((o) => o.name).includes(option.name);
+			return this.remoteParameterOptionsKeys.includes(option.name);
```

Comment:
It's more efficient to have `remoteParameterOptions` key as a computed and check if it includes `option.name` rather than mapping it every time for each option which leads in app hanging for several seconds if there are many options, for example, in CoinGeco node.